.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "\U0001F41E Bug report"
+about: Let us know if something isn't working as you expect it to.
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**WordCamp:**
+If this is a problem on a specific WordCamp's site, list that here.
+
+**System (please complete the following information):**
```

Comment:
`MAYBE` - I'd remove this section, since it feels like a lot of work to fill it in w/ specific version numbers etc, and the vast majority of issues can be reproduced without it. It feels like it adds friction to the process without much benefit. We can always follow up and ask for this info if we can't reproduce something in the browsers we have installed locally.

Reply:
I'd rather leave it- I've had a lot of issues created where I follow up to ask for this info and the person never replies.

browser/OS is also more important in JS issues, so as we're building more JS-focused things, this will be more useful.

Reply:
In cases when you can't reproduce it locally in the current version of the major browsers, I would assume that the most likely cause is a browser extension. Is that true in your experience, or did most of the issues that came down to differences in supported versions of the browsers?

It seems like if we can't reproduce it, and the reporter doesn't care enough about it to respond, then we could just close it as `maybelater` and let another reporter re-open it if someone ever runs into it again.

Reply:
Having them report the browser short-circuits the need to trial-and-error tests among all browsers (start in Chrome, then try again in FF, maybe now try Safari, then boot the VM to try Windows + Chrome, Firefox, IE…).

Adding a note about trying with browser extensions disabled could be a good idea though.

Reply:
To me the amount of friction for the user and the dev feels about the same in those cases, but we'd be adding that friction for users every time in order to save us from that friction in a small percent of cases.

So, personally, it still feels like it hurts them more than it helps us.